tests/mock_data/calicoctl/explicit_peering/route_reflector/delete.yaml

```diff
@@ -0,0 +1,26 @@
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-1
+spec:
+  peerIP: 10.192.0.3
+  asNumber: 64566
+  node: kube-master
+
+---
+
+kind: IPPool
+apiVersion: projectcalico.org/v3
+metadata:
+  name: ippool-1
+spec:
+  cidr: 192.168.0.0/16
+  ipipMode: Always
+  natOutgoing: true
+
+---
+
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-v6
```

Comment:
Is this one intentionally truncated?

Reply:
Yes. I thought that only the metadata should be relevant for a deletion, so here I took the chance to check that that was correct. Is that OK?